Makefile

```diff
@@ -2519,9 +2517,9 @@ swift_deploy_cleanup: ## cleans up the service instance, Does not affect the ope
 
 ##@ CERT-MANAGER
 .PHONY: certmanager
-certmanager: export NAMESPACE=$(if $(findstring 4.10,$(OCP_RELEASE)),openshift-cert-manager,cert-manager)
-certmanager: export OPERATOR_NAMESPACE=$(if $(findstring 4.10,$(OCP_RELEASE)),openshift-cert-manager-operator,cert-manager-operator)
-certmanager: export CHANNEL=$(if $(findstring 4.10,$(OCP_RELEASE)),tech-preview,$(if $(findstring 4.15,$(OCP_RELEASE)),stable-v1.13,stable-v1.11))
+certmanager: export NAMESPACE=cert-manager
+certmanager: export OPERATOR_NAMESPACE=cert-manager-operator
+certmanager: export CHANNEL=stable-v1
```

Comment:
With crc with 4.15.2 when using stable-v1 (default install_yamls target` I noticed below error.

`  - message: '[failed to populate resolver cache from source certified-operators/openshift-marketplace:
      failed to list bundles: rpc error: code = Unavailable desc = connection error:
      desc = "transport: Error while dialing dial tcp 10.217.5.48:50051: connect:
      connection refused", failed to populate resolver cache from source community-operators/openshift-marketplace:
      failed to list bundles: rpc error: code = Unavailable desc = connection error:
      desc = "transport: Error while dialing dial tcp 10.217.4.125:50051: connect:
      connection refused", failed to populate resolver cache from source redhat-marketplace/openshift-marketplace:
      failed to list bundles: rpc error: code = Unavailable desc = connection error:
      desc = "transport: Error while dialing dial tcp 10.217.5.46:50051: connect:
      connection refused", failed to populate resolver cache from source redhat-operators/openshift-marketplace:
      failed to list bundles: rpc error: code = Unavailable desc = connection error:
      desc = "transport: Error while dialing dial tcp 10.217.4.170:50051: connect:
      connection refused"]'
    reason: ErrorPreventedResolution
    status: "True"
    type: ResolutionFailed
`
And it gets fixed with 

$ oc patch OperatorHub cluster --type json -p '[{"op": "add", "path": "/spec/disableAllDefaultSources", "value": true}]' 

and then 
$ oc patch OperatorHub cluster --type json -p '[{"op": "add", "path": "/spec/disableAllDefaultSources", "value": false}]' 

I saw this KB https://access.redhat.com/solutions/7052456 for something like that... Mentioning here for someone facing the same issue.